app/timetables/models.py

```diff
@@ -39,3 +39,20 @@ def clean(self):
 
     def __str__(self):
         return self.name
+
+
+class Dish(ForceCapitalizeMixin, TimestampMixin):
+    """
```

Comment:
@andela-joyebanji IMO the `ForceCapitalizeMixin` is not appropriate for this model. That mixin will not only capitalize the first letter, it will force every other letter in the name to lowercase. This might be okay in some cases but not in all e.g. 
'jollof rice and plantain' => 'Jollof rice and plantain'
'Moi-Moi and Quaker Oats' => 'Moi-moi and quaker oats'.  # note that Quaker Oats is brand name and should be capitalized.
Although we would like the `name` of dishes to be unique, we can ignore letter casing in this case since the names can get very complex.